### ISSUE TITLE
🦄 test split on Valkyrized Hyku

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/iiif_print#341]
# Summary

Test Splitting with the Valkyrized Hyku and make tickets for issues found.

**Comment by laritakr** [comment]
blocked by https://github.com/samvera/hyku/issues/2177

**Comment by laritakr** [comment]
Splitting and relationships jobs fail. Moving to a new ticket to fix them.
